wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1045,6 +1082,28 @@ public final class MLSService: MLSServiceInterface {
         }
     }
 
+    // MARK: - Create Conversation
+
+    enum MLSConversationError: Error, Equatable {
+        case failedToCreateConversation
+    }
+
+    func createConversation(for groupID: MLSGroupID) throws {
+
+        let config = ConversationConfiguration(
+            ciphersuite: .mls128Dhkemx25519Aes128gcmSha256Ed25519,
+            externalSenders: backendPublicKeys.ed25519Keys,
+            custom: .init(keyRotationSpan: nil, wirePolicy: nil)
+        )
+
+        do {
+            try coreCrypto.perform { try $0.createConversation(conversationId: groupID.bytes, config: config) }
+        } catch {
+            logger.error("failed to createConversation: \(String(describing: error))")
+            throw MLSConversationError.failedToCreateConversation
+        }
+    }
+
```

Comment:
Why do we need this? Can't we use `createGroup` instead?

Reply:
I don't know that's what the ticket said to call createConversation

Reply:
But the `createGroup` method already does this, it's basically the same method you've implemented here.

Reply:
🤦🏻‍♂️